src/scripts/browserDeviceProfile.js

```diff
@@ -719,17 +718,15 @@ import browser from './browser';
             }
         }
 
-        if (webmAudioCodecs.length && webmVideoCodecs.length) {
+        // Progressive mp4 transcoding
+        if (mp4VideoCodecs.length && videoAudioCodecs.length) {
             profile.TranscodingProfiles.push({
-                Container: 'webm',
+                Container: 'mp4',
                 Type: 'Video',
-                AudioCodec: webmAudioCodecs.join(','),
-                // TODO: Remove workaround when servers migrate away from 'vpx' for transcoding profiles.
-                VideoCodec: (canPlayVp8 ? webmVideoCodecs.concat('vpx') : webmVideoCodecs).join(','),
+                AudioCodec: videoAudioCodecs.join(','),
+                VideoCodec: mp4VideoCodecs.join(','),
                 Context: 'Streaming',
                 Protocol: 'http',
-                // If audio transcoding is needed, limit channels to number of physical audio channels
```

Comment:
Why did you remove this?

Reply:
`MaxAudioChannels: physicalAudioChannels.toString()`

This is already commonly used in most profiles above, so I think it's safe to remove the comment.